src/FileConventions.Test/FileConventions.Test.fs

```diff
@@ -670,3 +670,51 @@ let NamespaceConvention2() =
         ))
 
     Assert.That(NotFollowingNamespaceConvention fileInfo, Is.EqualTo true)
+
+
+[<Test>]
+let NamespaceConvention3() =
+    let fileInfo =
+        (FileInfo(
+            Path.Combine(
+                dummyFilesDirectory.FullName,
+                "src",
+                "Foo",
+                "Bar",
+                "DummyFileUnderBarWithRightNamespace.fs"
+            )
+        ))
+
+    Assert.That(NotFollowingNamespaceConvention fileInfo, Is.EqualTo false)
```

Comment:
Since these asserts, keep happening and I'm not sure which one is desired here, I'm not gonna mention them in the upcoming reviews for this PR. But, if one of them is changed to Assert.False or Assert.True all of them have to change.

Reply:
@Mersho please don't change